packages/@react-spectrum/provider/docs/Provider.mdx

```diff
@@ -0,0 +1,160 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/provider';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Button} from '@react-spectrum/button';
+import {Checkbox} from '@react-spectrum/checkbox';
+import {Form} from '@react-spectrum/form';
+import {Provider} from '@react-spectrum/provider';
+import scaleMedium from '@adobe/spectrum-css-temp/vars/spectrum-medium-unique.css';
+import {TextField} from '@react-spectrum/textfield';
+import themeLight from '@adobe/spectrum-css-temp/vars/spectrum-light-unique.css';
+```
+
+# Provider
+
+<p>{docs.exports.Provider.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Provider']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<Provider isDisabled scale="large">
+  <Button variant="primary">Button</Button>
+</Provider>
+```
+
+## Children Props
+
+As a convenience, Provider supports passing common props to all components
+within it. For example, an entire group of components could be disabled by
+setting `isDisabled` in one place rather than on each individual element.
+These props are `isDisabled`, `isEmphasized`, `isQuiet`, `isReadOnly`,
+`isRequired`, and `validationState`.
+
+### Quiet
+
+```tsx example
+<Provider isQuiet>
+  <Form>
+    <div>
+      <TextField label="Email" />
+    </div>
+    <Checkbox>
+      Checkbox Label
+    </Checkbox>
+    <div>
+      <Button variant="primary">Button</Button>
+    </div>
+  </Form>
+</Provider>
+```
+
+## Global Settings
+
+Props such as `theme`, `colorScheme`, `locale`, and `scale` adjust all
+components with Provider. The `theme` allows the definition of custom CSS
```

Comment:
Maybe "Props such as `theme`, `colorScheme`, `locale`, and `scale` are applied to all child components, including nested Providers"? Just trying to make the differentiation between the `Children Props` section and this one more clear

Reply:
bump on this suggestion, lemme know what you think